tmk_core/protocol/chibios/main.c

```diff
@@ -232,7 +232,7 @@ int main(void) {
                 suspend_power_down();  // on AVR this deep sleeps for 15ms
                 /* Remote wakeup */
                 if (suspend_wakeup_condition()) {
-                    usbWakeupHost(&USB_DRIVER);
```

Comment:
usbWakeupHost() must be used to send a resume signal to host (from J state to K state)

https://www.nxp.com/docs/en/application-note/AN5385.pdf

Reply:
A good question is, is this actually required, because, the lack of it doesn't seem to adversely affect the keyboard, nor it's ability to wake the system.